src/Spout/Writer/Style/Style.php

```diff
@@ -269,16 +269,25 @@ public function shouldWrapText()
     }
 
     /**
+     * @param boolean $shouldWrap Should the text be wrapped
```

Comment:
I have noticed that almost all PHPDoc in this file is written with 'boolean' instead of 'bool'